src/env/obfuscated.rs

```diff
@@ -84,11 +86,54 @@ impl ObfuscatedFileSystem {
         self.files.load(Ordering::Relaxed)
     }
 }
+pub struct ObfuscatedContext(<DefaultFileSystem as FileSystem>::AsyncIoContext);
+impl AsyncContext for ObfuscatedContext {
+    fn wait(&mut self) -> IoResult<usize> {
+        self.0.wait()
+    }
```

Comment:
These codes are not covered, meaning you didn't test `ObfuscatedFileSystem` in unit tests.